ui/src/main.rs

```diff
@@ -72,9 +75,19 @@ fn main() {
     let file_logger = FileLogger::new(logfile).expect("Unable to create file logger");
     let logger = StatisticLogger::new(file_logger);
     let rewrite = Rewrite::new(vec![vec!["help".into()]], "/index.html".into());
+    let cors = CorsMiddleware { 
+        allowed_origins : AllowedOrigins::Any { allow_null: false }, 
+        allowed_headers: vec![UniCase("Content-Type".to_owned())],
+        allowed_methods : vec![ Get, Post ],
+        exposed_headers: vec![],
+        allow_credentials: false,
+        max_age_seconds: 60 * 60,
```

Comment:
Just as a note for your crate, you might want to take a `std::time::Duration` here; then you wouldn't have to add `seconds` to the field name.